crates/types/src/traits/election.rs

```diff
@@ -30,6 +29,9 @@ pub trait Membership<TYPES: NodeType>:
     /// Get all participants in the committee for a specific view
     fn committee_members(&self, view_number: TYPES::Time) -> BTreeSet<TYPES::SignatureKey>;
 
+    /// Get all leaders in the committee for a specific view
+    fn committee_leaders(&self, view_number: TYPES::Time) -> BTreeSet<TYPES::SignatureKey>;
+
     /// Get the stake table entry for a public key, returns `None` if the
```

Comment:
Is this function just so we can run the static committee with a single leader, and know that leader?

Reply:
I saw it was just used to print the number of eligible leaders on creation

Reply:
yeah, this is just used to get the number of eligible leaders into `BenchResults` for benchmarking.